app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    new_comment = current_user.comments.new(
+      text: params.require(:comment).permit(:text)[:text],
```

Comment:
- [ ] Same case as above. Create a private method for the strong params to encapsulate them.

  Another issue, the new action is not implemented.
  
  Sugfestion:
  
  ```ruby
  def new
    @comment = Comment.new
  end
  
  def create
    @comment = @current_user.comments.new(comment_params)
    @comment.post_id = params[:post_id]
    # the rest of create action logic
  end
  
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```